vnext/Microsoft.ReactNative/ReactHost/React.h

```diff
@@ -229,6 +206,58 @@ struct ReactOptions {
 
   ReactDevOptions DeveloperSettings = {};
 
+  //! This controls the availiablility of various developer support functionality including
+  //! RedBox, and the Developer Menu
+  void SetUseDeveloperSupport(bool enable) noexcept;
+  bool UseDeveloperSupport() const noexcept;
+  static void SetUseDeveloperSupport(
+      winrt::Microsoft::ReactNative::IReactPropertyBag const &properties,
+      bool value) noexcept;
+  static bool UseDeveloperSupport(winrt::Microsoft::ReactNative::IReactPropertyBag const &properties) noexcept;
+
+  //! Should the instance trigger the hot reload logic when it first loads the instance
+  //! Most edits should be visible within a second or two without the instance having to reload
+  //! Non-compatible changes still cause full reloads
+  void SetUseFastRefresh(bool enable) noexcept;
+  bool UseFastRefresh() const noexcept;
+  static void SetUseFastRefresh(
```

Comment:
I wonder if it would be cleaner to have a single IInspectable DevOptions struct in the property bag instead of helpers to set each separately.